integration/tests/cook/test_client.py

```diff
@@ -115,6 +115,7 @@ def test_container_port_submit(self):
         uuid = self.client.submit(command=f'{hostname_progress_cmd} && nc -l -p {JOB_PORT} $(hostname -I)',
                                   container=container,
                                   env={progress_file_env: 'progress.txt'},
+                                  max_retries=5,
```

Comment:
This test fix is unrelated to the rest of the PR, but the test failed on this branch, so I added this here.